exocom-server/src/websocket-subsystem/websocket-subsystem.ls

```diff
@@ -65,6 +68,7 @@ class WebSocketSubsystem extends EventEmitter
   # Registers the given websocket as a connection
   # to an instance of the service with the given name
   register-client: ({client-name, websocket}) ->
+    @session-ids[client-name] = uuid.v1!
```

Comment:
Session ids will be passed in the message and are created only on the websocket bridge. Exocom should not be creating them. I had envisioned this PR only touching the exorelay implementations.